### ISSUE TITLE
Method reader fails to process call of method that has multiple args with numeric conversion in case non-binary wire is used

### ISSUE THREAD

**Issue by glukos** [OpenHFT/Chronicle-Wire#208]
Reproducer test:
```
    public interface HasCallWithTwoConversionArgs {
        void call(@LongConversion(Base64LongConverter.class) long l, @IntConversion(Base32IntConverter.class) int i);
    }

    @Test
    public void testTextWireArgsNumericConversion() {
        Wire wire = new TextWire(Bytes.allocateElasticOnHeap(256));

        HasCallWithTwoConversionArgs methodWriter = wire.methodWriter(HasCallWithTwoConversionArgs.class);
        methodWriter.call(-426L, 70);

        AtomicLong readLongValue = new AtomicLong();
        AtomicInteger readIntValue = new AtomicInteger();

        HasCallWithTwoConversionArgs impl = (l, i) -> {
            readLongValue.set(l);
            readIntValue.set(i);
        };

        MethodReader methodReader = wire.methodReader(impl);

        methodReader.readOne();

        assertEquals(-426L, readLongValue.get());
        assertEquals(70, readIntValue.get());
    }
```
This issue is not critical though as main purpose of numeric conversion annotations is to mark DTO fields, not method arguments.

**Comment by peter-lawrey** [comment]
Currently, using a conversion on more than one argument isn't supported atm, this should really be a DTO.
However, good to include here so we can support it in the future.

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.20.66](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.20.66), [BOM-2.20.86](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.86)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.20.101](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.20.101), [BOM-2.20.134](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.134)
